src/media/css/elements--categories.styl

```diff
@@ -34,6 +36,9 @@ mkt-category-item {
     padding: 10px;
     transition: none;
 
+    +media--wide-tablet() {
```

Comment:
I prefer it closer to the base styles like this. I thought we had decided to use this for bidi/responsive stuff now.